en/components/general/wpf_to_angular_guide/create_first_angular_app.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Create your first Angular app
+_description: WPF to Angular guide topics help you switch from WPF to Angular.
+_keywords: WPF, Windows Presentation Foundation, Angular, Ignite UI for Angular
+---
+
+## Create your first Angular app
+
+If you are a WPF developer, creating new projects inside of Visual Studio is pretty straight forward. You would just click File -> New Project, select the project type, give it a name and press OK. Since you are going into the Angular world, you want to create a new project inside Visual Studio Code. However, there is no new project option here and that's because Visual Studio Code is file based and not project based. In order to create a new Angular application, we are going to use the command prompt.
+
```

Comment:
We haven't added such because we have some prerequisites in the [Getting Started](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/getting_started.html) topic, but probably we could add them here as well.

Reply:
I have added **Prerequisites** section.